README.md

```diff
@@ -36,58 +43,71 @@ module "example" {
 | Name | Version |
 |------|---------|
 | terraform | ~> 1.0 |
-| aws | ~> 4.9 |
+| aws | ~> 5.0 |
 
 ## Providers ##
 
 | Name | Version |
 |------|---------|
-| aws | ~> 4.9 |
+| aws.provisionaccount | ~> 5.0 |
 
 ## Modules ##
 
-No modules.
+| Name | Source | Version |
+|------|--------|---------|
+| aws\_key\_pair | cloudposse/key-pair/aws | 0.18.3 |
+| documentdb-cluster | cloudposse/documentdb-cluster/aws | 0.27.0 |
+| ec2 | cloudposse/ec2-instance/aws | 1.6.0 |
+| kevsync\_eventbridge | terraform-aws-modules/eventbridge/aws | 3.11.0 |
+| kevsync\_lambda | terraform-aws-modules/lambda/aws | 7.9.0 |
+| subnets | cloudposse/dynamic-subnets/aws | 2.4.2 |
+| vpc | cloudposse/vpc/aws | 2.1.1 |
 
 ## Resources ##
 
 | Name | Type |
 |------|------|
-| [aws_instance.example](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/instance) | resource |
-| [aws_ami.example](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/ami) | data source |
-| [aws_default_tags.default](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/default_tags) | data source |
+| [aws_security_group_rule.egress_from_ec2_to_documentdb](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group_rule) | resource |
+| [aws_security_group_rule.ingress_from_ec2_to_documentdb](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group_rule) | resource |
+| [aws_caller_identity.cyhy](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/caller_identity) | data source |
 
 ## Inputs ##
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
-| ami\_owner\_account\_id | The ID of the AWS account that owns the Example AMI, or "self" if the AMI is owned by the same account as the provisioner. | `string` | `"self"` | no |
-| aws\_availability\_zone | The AWS availability zone to deploy into (e.g. a, b, c, etc.). | `string` | `"a"` | no |
-| aws\_region | The AWS region to deploy into (e.g. us-east-1). | `string` | `"us-east-1"` | no |
-| subnet\_id | The ID of the AWS subnet to deploy into (e.g. subnet-0123456789abcdef0). | `string` | n/a | yes |
+| aws\_availability\_zones | The list of AWS availability zones to deploy into (e.g. ["us-east-1a", "us-east-1b", "us-east-1c"]. | `list(string)` | ```[ "us-east-1a", "us-east-1b", "us-east-1c" ]``` | no |
```

Comment:
Would it be better to just pull a list of _all_ AZs for the given region, as is done [here](https://github.com/cisagov/distributed-subnets-tf-module/blob/66879e8d2e19348160cccc212ec92e8fe1eabd34/azs.tf#L1-L10)?  Note that in that example we go on to sort the AZs since they are returned in a random order.

Reply:
I think for now, it's fine to let the user specify which AZs they want to use.  There may be cases where you don't want to use every AZ in a region (e.g. for $$$ reasons) or if you want to avoid certain AZs for some reason.